openpype/hosts/hiero/api/menu.py

```diff
@@ -138,3 +140,33 @@ def menu_install():
     exeprimental_action.triggered.connect(
         lambda: host_tools.show_experimental_tools_dialog(parent=main_window)
     )
+
+
+def add_scripts_menu():
+    try:
+        from . import launchforhiero
+    except ImportError:
+
+        log.warning(
+            "Skipping studio.menu install, because "
+            "'scriptsmenu' module seems unavailable."
+        )
+        return
+
+    # load configuration of custom menu
+    project_settings = get_project_settings(os.getenv("AVALON_PROJECT"))
+    config = project_settings["hiero"]["scriptsmenu"]["definition"]
+    _menu = project_settings["hiero"]["scriptsmenu"]["name"]
+
+    if not config:
+        log.warning("Skipping studio menu, no definition found.")
+        return
+
+    # run the launcher for Maya menu
+    studio_menu = launchforhiero.main(title=_menu.title())
+
+    # apply configuration
+    studio_menu.build_from_configuration(studio_menu, config)
+
+
+add_scripts_menu()
```

Comment:
Never call functions in global scope. This should be moved to function `install` in `pipeline.py`.